src/generators/date/index.ts

```diff
@@ -13,9 +13,15 @@ const createDateGenerator = (chance: Chance.Chance) => (
   from?: Date | string,
   to?: Date | string,
 ): Date => {
```

Comment:
Could you please add test coverage for the changes in this PR? (But please don't add to the `src/index.test.ts` file – add test files next to date and integer.)

Reply:
Added in 7211519